conf/groups.yml

```diff
@@ -216,6 +216,7 @@ Body:
       hack_info: true
       any_warp: true
       view_hpmeter: true
+      macro_detect: true
```

Comment:
Causes crash on windows when running map-server it stops in groups.yml [6/8]

Reply:
We are working on the feedback already. Wait a little before posting further feedback and testing.

Reply:
Me to.